src/Components/SearchBarDetails.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+/**
+ * Representing the details from the searched character
+ */
+const SearchBarDetails = ({ searchedItemDetails, showSearchBarDetails }) => {
+
+    var divStyle = {
+        fontSize: '18px',
+        padding:0
+        };
+
+    let detailItems = Object.keys(searchedItemDetails).map((key, index) => {
+        return <li key={index}> {[searchedItemDetails[key]]}</li>;
```

Comment:
If the list changes dynamically or items are to be deleted, having an index as a key is a [bad idea. ](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318)

Reply:
I agree, but then what should I use as a key since there isn't a unique ID in the response from the API?

Reply:
If they don't change, index is fine, if you plan to re-arrange or delete some list items, then they should be unique.